src/content/ui-components/installation/next.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Next.js
+meta:
+  title: Next.js | Tiptap UI Component
+  description: Install and configure Tiptap UI Components in your Next.js project.
+  category: UI Components
+sidebars:
+  hideSecondary: true
+---
+
+import { Callout } from '@/components/ui/Callout'
+
+### Create a New Next.js Project
+
+Run the `init` command to create a new Next.js project or to setup an existing one:
+
+```bash
+pnpm dlx @tiptap/cli init
```

Comment:
Always provide installation commands as npm commands in the Tiptap docs. Not pnpm.

Later on I'd like to add a package manager switcher.